contracts/pol/CarbonPOL.sol

```diff
@@ -154,6 +192,20 @@ contract CarbonPOL is ICarbonPOL, Upgradeable, ReentrancyGuardUpgradeable, Utils
         return _priceDecayHalfLife;
     }
 
+    /**
+     * @inheritdoc ICarbonPOL
+     */
+    function ethSaleAmount() external view returns (uint128) {
+        return _ethSaleAmount.initial;
+    }
+
+    /**
+     * @inheritdoc ICarbonPOL
+     */
+    function currentEthSaleAmount() external view returns (uint128) {
```

Comment:
I'd consider returning the `EthSaleAmount memory` directly instead of splitting it into two functions

Reply:
IMHO, as someone who interacts with contracts all the time, both options are equivalvently user friendly. Every modern framework (wagmi, ethers, viem, etc.) handles input/ouput structs very easily and convinently.

Reply:
Changed - personally, I'm not a fan of the `carbonPOL.ethSaleAmount().initial` syntax.